pydiggy/node.py

```diff
@@ -563,7 +563,7 @@ def _explode(
         include: List[str] = None,
     ) -> Dict[str, Any]:
         """
-        Explode a Node object into a mapping 
+        Explode a Node object into a mapping

```

Comment:
My environment automatically removed this little bit of white space. I can remove the change if you want to keep the commit history clean.